src/Typed/.gitattributes

```diff
@@ -0,0 +1,6 @@
+/.gitattributes export-ignore
+/.gitignore export-ignore
+/Resources/assets/test export-ignore
+/Animation.gif export-ignore
+/CHANGELOG.md export-ignore
+/README.md export-ignore
```

Comment:
No reason for these last 2. These might be useful for someone looking through the code.

Reply:
I usually list all non-code-related files in `.gitattributes`. Bad habits die hard 😬.